### PR TITLE
ci: Add workflow for publishing extension

### PR DESCRIPTION
**Pull request by SevInf**
npm package would autoamtically get published when new tag is created.
Tested on a fork.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          version: 8
+
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: 'https://registry.npmjs.org'
+          node-version: 16
+          cache: 'pnpm'
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
```
Comment (Jolg42) [inline .github/workflows/publish.yml:25]:
```suggestion
        run: pnpm install
```

Not needed, it's the default in CI when a lockfile is present

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+      - name: Version
+        run: pnpm version --no-git-tag-version ${{ github.event.release.tag_name }}
+
+      - name: Publish
+        run: |
+          pnpm config set "//registry.npmjs.org/:_authToken" "${NPM_TOKEN}"
```
Comment (Jolg42) [inline .github/workflows/publish.yml:32]:
```suggestion
```
Not needed, it's already done via 
```
      - uses: actions/setup-node@v3
        with:
          registry-url: 'https://registry.npmjs.org'
```

**Review by Jolg42 — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 2 above)
Comment (SevInf) [reply .github/workflows/publish.yml:33]: `setup-node` one, unfrotinately, is not picked up by `pnpm`

Comment (Jolg42) [reply .github/workflows/publish.yml:33]: That's very weird, I don't see why that would not be picked up

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+name: Publish (latest)
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply .github/workflows/publish.yml:1]: Didn't like it? 😄

**Pull request merged**